sdk/communication/communication-chat/src/chatThreadClient.ts

```diff
@@ -61,8 +70,8 @@ export class ChatThreadClient {
   private timeOfLastTypingRequest: Date | undefined = undefined;
 
   constructor(
-    threadId: string,
     private readonly url: string,
```

Comment:
We got feedback that we should rename `url` to `endpoint` in the clients

Reply:
I will put these review changes in preview 5.